invenio_banners/services/service.py

```diff
@@ -34,25 +34,6 @@ def read(self, identity, id):
             links_tpl=self.links_item_tpl,
         )
 
-    def read_active_banners(self, identity, url_path):
-        """Retrieve the list of active banners with the given url_path."""
-        self.require_permission(identity, "read")
-
-        active_banners = self.record_cls.get_active()
-
-        # filter by url_path
-        banners = []
-        for banner in active_banners:
-            if banner.url_path is None or url_path.startswith(banner.url_path):
-                banners.append(banner)
-
-        return self.result_list(
-            self,
-            identity,
-            banners,
-            links_item_tpl=self.links_item_tpl,
-        )
```

Comment:
Just by curiosity. Why removing this function?

Reply:
Moved the filtering to the db layer, because it's easier to access from utils `get_active_banners_for_request` (I had some issuer with accessing service methods from there)